docker-compose.yml

```diff
@@ -13,3 +24,30 @@ services:
     environment:
       # - EMAIL=foo@bar.org
       # - NOTIFY='tgram://...'
+      # - NOTIFY_TITLE="Optional title for notifications"
```

Comment:
Don't want another place to keep options in sync besides `config.js` and `README.md`.
Is this useful for some software to offer a template? (There's already some external Unraid template.)

Reply:
That is a good wish. I can remove them, but at least compose example should be copy-paste working, so that it contains basic needed data.